docs/workflows.rst

```diff
@@ -119,7 +119,7 @@ Framewise displacement calculation and thresholding
 :class:`~xcp_d.interfaces.censoring.GenerateConfounds`,
 :func:`~xcp_d.utils.modified_data.compute_fd`
 
-Framewise displacement is then calculated according to the formula from Power et al. (CITE).
+Framewise displacement is then calculated according to the formula from Power et al. :footcite:t:`power_fd_dvars`.
```

Comment:
```suggestion
Framewise displacement is then calculated according to the formula from :footcite:t:`power_fd_dvars`.
```

Reply:
The `:t:` (instead of `:p:`) means that it's an in-text citation, so we don't need the authors beforehand.